Nos/Views/NoteComposer/NoteComposer.swift

```diff
@@ -201,15 +227,40 @@ struct NoteComposer: View {
         }
         isPresented = false
     }
+    
+    private func loadQuotedNote() {
+        guard let quotedNoteID else {
+            return
+        }
+        
+        @Dependency(\.persistenceController) var persistenceController
```

Comment:
In general, I like to have dependencies defined as properties near the top of the type, though I know we don't always do that. Would you mind moving this up with the other dependencies?

Reply:
I agree